content/blog/2023-04-01-etc-is-migrating-to-pos-sharding-its-blockchain-and-lifting-its-supply-cap/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "ETC Is Migrating to PoS, Sharding its Blockchain, and Lifting its Supply Cap"
+date: 2023-04-01
+author: Donald McIntyre
+contributors: ["DonaldMcIntyre"]
+tags: ["announcement"]
+linkImage: ./etc-april-fools.png
+---
+
+## Proof of Stake Has Proven to be Better
+
+After 6 months of observing the evolution of the Ethereum blockchain in its proof of stake consensus mechanism, the Ethereum Classic community has voted unanimously to migrate ETC to that system.
+
+The top reasons for reaching this unprecedented level of agreement amongst community members were:
+
+- Proof of stake clearly advances ETC to its most valued goal of social consensus.
+- Proof of stake reduces the consumption of electricity, saving trees, polar bears, bees, and, above all, the Kamchatka Peninsula brown ant, a species very high in the UN's protected species list.
+- Ethereum has proven the benefits of becoming a security, with very close ties with regulators and the World Economic Forum, to accelerate its agenda of non-tyrannical on-chain passport, identity systems, and intelligence agency backdoors.
+
+Istora, a prominent Japanese ETC community contributor, said "centralization and regulator friendliness has been a long (though little known) goal of ETC for a long time. Proof of stake establishes the basis for this highly sought objective."
+
+## Sharding Is Expected in ETC Even Before Ethereum
+
+It has been proven by cloud services and corporate data centers that sharding of the database and eliminating the cumbersome full replication of blockchain information is an incredibly effective (and secure) method to achieve near-infinite scaling.
+
+The ETC community has voted unanimously to approve the sharding of its blockchain to enable a one-dapp-per-shard model. This will make Ethereum Classic infinitely scalable in a matter of weeks. This is because the ETC community has decided to integrate sharding ASAP to leap over Ethereum, which will likely take another 25 or 30 years to implement the revolutionary technology.
+
+Bob Summerwill, ETC's CEO, said that sharding is a key milestone for Ethereum Classic to achieve ossification, and that walking at least 10,000 steps per day is very good to reverse osteoporosis.
+
+## Recent Bank Bailouts Pushes the Community to Lift the Supply Cap
+
+After the nerve wrecking days of the recent financial crisis, the ETC community has come to the realization that a fixed supply and an algorithmic monetary policy is not only evil but one of the worse ways to protect people's property on a global scale. The best method is to be able to print money whenever the community deems necessary, bringing four solutions to the world at the same time, with no cost whatsoever to ETC constituents:
```

Comment:
Line 32: Correct spelling; Correct usage; Correct pronoun usage; Correct subject-verb agreement; Edit for agreement
```suggestion
After the nerve-wracking days of the recent financial crisis, the ETC community has come to the realization that a fixed supply and an algorithmic monetary policy are not only evil but are among the worst ways to protect people's property on a global scale. The best method is to be able to print money whenever the community deems it necessary, bringing four solutions to the world at the same time, with no cost whatsoever to ETC constituents:
```